docs/src/howto-guides/008_config_local_mqtt_port.md

```diff
@@ -50,8 +56,19 @@ Below example shows that we can not set string value for the port number.
 ```shell
 tedge config set mqtt.port '"1024"'
 
-Error: failed to set the configuration key: mqtt.port with value: "1234".
+Error: failed to set the configuration key: mqtt.port with value: "1024".
```

Comment:
Not sure if it is a good idea to use 1024 as this is the first  (or last) that can be used without elevated privledges

Reply:
ok changed it to 1234